addons/pos_restaurant/tests/test_frontend.py

```diff
@@ -278,3 +278,7 @@ def test_11_bill_screen_qrcode(self):
     def test_12_merge_table(self):
         self.pos_config.with_user(self.pos_admin).open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.pos_config.id, 'MergeTableTour', login="pos_admin")
+
+    def test_13_duplicate_floor(self):
+        self.pos_config.with_user(self.pos_admin).open_ui()
+        self.start_tour("/pos/ui?config_id=%d" % self.pos_config.id, "DuplicateFloorTour", login="pos_admin")
```

Comment:
can you move the steps from this 2 tests in the `FloorScreenTour` ? This way we'll have one tour forall the floor screen features